### ISSUE TITLE
File upload elements order confusion

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1182]
Success Criteria 1.3.2 in WCAG 2 fails due confusing order of uploaded file elements, which is quite bad for a screenreader in particular:

* uploaded files are placed above the upload section (handled by the `cradmin-legacy-bulkfileupload-progress-item cradmin-legacy-bulkfileupload-progress-item-finished` class) which makes it unclear for screenreader users if the file(s) got uploaded, especially since forward navigation doesn't display the uploaded files.

**Comment by espenak** [comment]
Done, but waiting for cradmin-legacy release.
